dotnet/notebooks/10-BingSearch-using-kernel.ipynb

```diff
@@ -83,8 +83,8 @@
    },
    "outputs": [],
    "source": [
-    "using Microsoft.SemanticKernel.Skills.Web;\n",
-    "using Microsoft.SemanticKernel.Skills.Web.Bing;"
+    "Microsoft.SemanticKernel.Plugins.Web;\n",
+    "Microsoft.SemanticKernel.Plugins.Web.Bing;"
```

Comment:
Missing usings, are they needed ?
```suggestion
    "using Microsoft.SemanticKernel.Plugins.Web;\n",
    "using Microsoft.SemanticKernel.Plugins.Web.Bing;"
```

Reply:
Removing notebook changes, they shouldn't be part of this PR